package.json

```diff
@@ -38,6 +38,7 @@
     "@mui/x-date-pickers": "^6.5.0",
     "@next/font": "^13.0.5",
     "@paypal/react-paypal-js": "^7.8.1",
+    "@ramonak/react-progress-bar": "^5.0.3",
```

Comment:
Why is a new progress bar package needed?

Reply:
I was not able to achieve this effect: 
![image](https://github.com/podkrepi-bg/frontend/assets/14351733/a5ec1cbc-5526-4544-bddc-14d163c680a9)

 with the Progress bar provided from MUI (https://mui.com/material-ui/react-progress/).

Reply:
I had the same question but, now that we have the answer - all good. Later we can remove it if we find a way to do it without separate library.